Codeinwp/otter-blocks
# Animation Optimization

Closes #1033.


### Summary


This adds a new option to Dashboard (enabled by default), which toggles how we load animations in the front-end. If it's enabled, we only load CSS for the animations which are loaded on the page, not the entire file.

### Screenshots 

----

### Test instructions


You can disable/enable the option and ensure the animations file is only loaded when the option is disabled, and the animations continue to work either way properly.

For testing, you will have to use the Regenerate Styles option after you toggle. In code, we will force that option during the update migration.



---- 

### Checklist before the final review
